families/settings/Dockerfile-installed-xenial

```diff
@@ -25,9 +25,11 @@ RUN apt-get update \
  && apt-get install -y \
  curl \
  gcc \
+ git \
```

Comment:
Why is Git needed? Is it used in a script not part of this PR?

Reply:
The bin/get_version script runs `git describe` to determine the real version of the repository.